php/public/toggle-dark-mode.js

```diff
@@ -0,0 +1,26 @@
+// Function to toggle theme
+function toggleTheme() {
+    const currentTheme = document.documentElement.getAttribute('data-theme');
+    const newTheme = (currentTheme === 'dark') ? 'light' : 'dark';
```

Comment:
Would it be possible to add an 'auto' option? Sync with browser theme?

Reply:
Lets keep the manual mode for the first iteration. We can think about improvements in follow-ups